CHANGES.txt

```diff
@@ -15,6 +15,8 @@ Unreleased
 
 - SQLAlchemy: Fix SQL statement caching for CrateDB's ``OBJECT`` type.
 
+- SQLAlchemy: Refactor ``OBJECT`` type to use SQLAlchemy's JSON type machinery.
```

Comment:
I'd personally prefer `infrastructure` or `functionality` over `machinery`.

Reply:
Thanks. Updated the changelog item.